src/cljs/rems/profile.cljs

```diff
@@ -44,6 +46,29 @@
             :error-handler (flash-message/default-error-handler :top description)}))
    {}))
 
+(rf/reg-event-fx
+ ::generate-api-key
+ (fn [{:keys [db]} _]
+   (let [description [text :t.profile/generate-api-key]]
+     (post! "/api/user-settings/generate-ega-api-key"
+            {:params (::form db)
+             :handler (flash-message/default-success-handler :top description
+                                                             #(rf/dispatch [::user-settings]))
+             :error-handler (flash-message/default-error-handler :top description)}))
+   {}))
+
+(rf/reg-event-fx
+ ::delete-api-key
+ (fn [{:keys [db]} _]
+   (let [description [text :t.profile/delete-api-key]]
+     (post! "/api/user-settings/delete-ega-api-key"
+            {:params (::form db)
```

Comment:
what's in the form?

Reply:
Actually probably nothing for this. Seems like copypasta.